bin/storm.py

```diff
@@ -94,7 +94,7 @@ def init_storm_env():
 CONFFILE = ""
 JAR_JVM_OPTS = shlex.split(os.getenv('STORM_JAR_JVM_OPTS', ''))
 JAVA_HOME = os.getenv('JAVA_HOME', None)
-JAVA_CMD = 'java' if not JAVA_HOME else os.path.join(JAVA_HOME, 'bin', 'java')
+JAVA_CMD = 'java' if not JAVA_HOME else os.path.join(JAVA_HOME, 'bin', 'java' if not is_windows() else "java.exe")
```

Comment:
@srdo 
I meant to say we should set JAVA_CMD to `java.exe` for windows if JAVA_HOME is not provided. The line sets JAVA_CMD to `java` if JAVA_HOME is not provided regardless of OS.
Yes I think previous implementation itself is wrong.